apps/frontend/src/layouts/default.vue

```diff
@@ -533,6 +533,7 @@
         <a rel="noopener" :target="$external()" href="https://crowdin.com/project/modrinth">
           Crowdin
         </a>
+        <a rel="noopener" :target="$external()" href="https://bsky.app/profile/modrinth.com"> Bluesky </a>
```

Comment:
Looks a bit weird since that specific column is extended down with one extra element, while the others aren't. Perhaps we could replace one of current links? Regardless would move it up one so it sits next to X and Mastodon, not separated by Crowdin.

![image](https://github.com/user-attachments/assets/7eb1505b-ea00-4a2b-8cf3-9dbdf9651a65)